### PR TITLE
Add BitKeep Wallet

### PR DESCRIPTION
**Pull request by bitkeepwallet**
## Description

Add BitKeep Wallet to the page "finds wallet"
https://ethereum.org/en/wallets/find-wallet/

## Related Issue

BitKeep website: https://bitkeep.com

### CODE REVIEW

**Comment by minimalsm** [comment]
Hey @bitkeepwallet, thanks for taking the time to put up a PR for this.

Could you fill out the [wallet issue template](https://github.com/ethereum/ethereum-org-website/issues/new?assignees=&labels=Type%3A+Feature%2C+Type%3A+Content&template=suggest_wallet.md&title=) so the team have all the information required to look into your request properly? Thanks! 😬

**Comment by bitkeepwallet** [comment]
> Hey @bitkeepwallet, thanks for taking the time to put up a PR for this.
> 
> Could you fill out the [wallet issue template](https://github.com/ethereum/ethereum-org-website/issues/new?assignees=&labels=Type%3A+Feature%2C+Type%3A+Content&template=suggest_wallet.md&title=) so the team have all the information required to look into your request properly? Thanks! 😬

This is the issue. Thanks very much.
https://github.com/ethereum/ethereum-org-website/issues/3863

**Comment by bitkeepwallet** [comment]
Hi, please handle this, thanks very much.

**Comment by minimalsm** [comment]
Hi @bitkeepwallet thanks for this. Two things I wanted to note:

> BitKeep is the world's largest digital currency wallet, which supports all mainstream public chains and Layer 2 such as BTC, ETH, BSC, etc.

1. I'm not crazy about how this is phrased...
- Layer2s should be pluralised 
- We shouldn't be using the ticker (ETH) to refer to the blockchain (Ethereum)
- I don't think it's ethereum.org's job to inform users looking for an Ethereum wallet about alternative chains
- I don't think we should be claiming one blockchain to be more 'mainstream' than another 

With that in mind I'd be happier if we could rephrase this to something like this: `BitKeep is the world's largest digital currency wallet, which supports all major blockchains and Layer 2s.`

> the world's largest digital currency wallet

2. Not saying this is incorrect but can you substantiate these claims?

**Comment by bitkeepwallet** [comment]
> Hi @bitkeepwallet thanks for this. Two things I wanted to note:
> 
> > BitKeep is the world's largest digital currency wallet, which supports all mainstream public chains and Layer 2 such as BTC, ETH, BSC, etc.
> 
> 1. I'm not crazy about how this is phrased...
> 
> * Layer2s should be pluralised
> * We shouldn't be using the ticker (ETH) to refer to the blockchain (Ethereum)
> * I don't think it's ethereum.org's job to inform users looking for an Ethereum wallet about alternative chains
> * I don't think we should be claiming one blockchain to be more 'mainstream' than another
> 
> With that in mind I'd be happier if we could rephrase this to something like this: `BitKeep is the world's largest digital currency wallet, which supports all major blockchains and Layer 2s.`
> 
> > the world's largest digital currency wallet
> 
> 1. Not saying this is incorrect but can you substantiate these claims?

Thank you very much, your suggestion is very important to us, we have arranged related colleagues to re-adjust the description.

**Comment by minimalsm** [comment]
Thanks, @bitkeepwallet, it is good to go pending the description change. To have this up quickly we can start with the English description first and add translations versions later?

**Comment by bitkeepwallet** [comment]
> Thanks, @bitkeepwallet, it is good to go pending the description change. To have this up quickly we can start with the English description first and add translations versions later?

We have changed the all versions. The en is "BitKeep is a decentralized multi-chain digital wallet, dedicated to providing safe and convenient one-stop digital asset management services to users around the world. "

Please check. thanks very much.

**Review by jley81 — APPROVED** [review]
I approve merging these changes.

**Comment by minimalsm** [comment]
@bitkeepwallet last thing: could you tell me which of these translations have been machine-translated (e.g. google translate) and which are human translations? We run a translation program and should add anything that hasn't been human translated to our translation backlog.

**Comment by bitkeepwallet** [comment]
> @bitkeepwallet last thing: could you tell me which of these translations have been machine-translated (e.g. google translate) and which are human translations? We run a translation program and should add anything that hasn't been human translated to our translation backlog.

English and Chinese version are human translation. Others are machine-translated with Google Translate.
Thanks.
